packages/dev/core/src/Misc/fileTools.ts

```diff
@@ -357,7 +358,7 @@ export const ReadFile = (
 // eslint-disable-next-line @typescript-eslint/naming-convention
 export const LoadFile = (
     fileOrUrl: File | string,
-    onSuccess: (data: string | ArrayBuffer, responseURL?: string) => void,
+    onSuccess: (data: string | ArrayBuffer, responseURL?: string, contentType?: Nullable<string> | undefined) => void,

```

Comment:
```suggestion
    onSuccess: (data: string | ArrayBuffer, responseURL?: string, contentType?: string) => void,
```

I guess this would be enough, no?

Reply:
The Nullable is required but you're right there's no need for the undefined 😅